src/server/conn.rs

```diff
@@ -406,9 +406,13 @@ impl<E> Http<E> {
         self
     }
 
-    /// Sets whether HTTP2 is required.
+    /// Set the HTTP/2 maximum send buffer size.
     ///
-    /// Default is false
+    /// Default is ~ 400kb.
+    ///
+    /// # Panics
+    ///
+    /// The minimum value allowed is 8192. This method panics if the passed `max` is less than the minimum.
```

Comment:
I think you need to keep the original documentation here.

Reply:
You are correct, and I have no damn clue how I ended up changing the wrong docs hah.